recipes/steem/meta.yaml

```diff
@@ -0,0 +1,97 @@
+{% set name = "steem" %}
+{% set version = "0.18.103" %}
+{% set sha256 = "215A3A0E87151428AB71AB13FBEAB353F1FFF3F1A3EB082C5A4D3B2C7581D5D5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/07/7e/142fb6db1e14db39e658832b9d1b7f493d2e856ecd9c17f08490f85bf71e/steem-{{ version }}.tar.gz
+  sha256: {{ sha256|lower }}
+
+build:
+  number: 0
+  skip: True  # [py<36]
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - sphinx
+    - autopep8
+    - yapf
+    - sphinxcontrib-restbuilder
+    - sphinxcontrib-programoutput
+    - recommonmark
+    - pypandoc
+    - pytest-pylint
+    - pep8
+
+  run:
+    - python
+    - setuptools
+    - scrypt
+    - appdirs
+    - certifi
+    - ecdsa
+    - pycrypto
+    - toolz
+    - urllib3
+    - w3lib
+    - funcy
+    - langdetect
+    - prettytable
+    - voluptuous
+
+test:
+  imports:
+    - steem
+    - steem.account
+    - steem.amount
+    - steem.post
+    - steem.dex
+    - steem.aes
+    - steem.block
+    - steem.blockchain
+    - steem.cli
+    - steem.commit
+    - steem.converter
+    - steem.instance
+    - steem.profile
+    - steem.steem
+    - steem.steemd
+    - steem.transactionbuilder
+    - steem.utils
+    - steem.wallet
+    - steem.witness
+    - steembase
+    - steembase.account
+    - steembase.base58
+    - steembase.bip38
+    - steembase.chains
+    - steembase.exceptions
+    - steembase.http_client
+    - steembase.operationids
+    - steembase.operations
+    - steembase.storage
+    - steembase.transactions
+    - steembase.types
+  commands:
```

Comment:
You might consider testing the CLI as well (just make sure that running the help doesn't blow things up)